cmd/pipelines/deploy.go

```diff
@@ -44,11 +44,12 @@ func deployCommand() *cobra.Command {
 					b.Config.Bundle.Deployment.FailOnActiveRuns = failOnActiveRuns
 				}
 			},
-			Verbose:      verbose,
-			AlwaysPull:   true,
-			FastValidate: true,
-			Build:        true,
-			Deploy:       true,
+			Verbose:        verbose,
+			AlwaysPull:     true,
+			FastValidate:   true,
+			Build:          true,
+			Deploy:         true,
+			IsPipelinesCLI: true,
```

Comment:
This scopes it down to deploy, but it seems this is something that would be useful in plan & validate as well, no?